src/api/controllers/guilds.ts

```diff
@@ -52,6 +52,14 @@ export async function handleInternalGuildDelete(data: DiscordPayload) {
     }
   });
 
+  cacheHandlers.forEach("members", (member) => {
+    member.guilds.delete(payload.id);
+    
+    if (!member.guilds.size) await cacheHandlers.delete("members", member.id);
```

Comment:
Add a return after this else the member will be cached again

Reply:
:stuck_out_tongue: forgot to add that